src/platform/packages/shared/kbn-unified-data-table/src/components/data_table_columns.tsx

```diff
@@ -14,6 +14,7 @@ import {
   type EuiDataGridColumnCellAction,
   EuiScreenReaderOnly,
   EuiListGroupItemProps,
+  EuiDataGridColumnSortingConfig,
```

Comment:
```suggestion
  type EuiDataGridColumnSortingConfig,
```
Maybe this would reduce the bundle size a bit?

Reply:
Let's see, updated in: https://github.com/elastic/kibana/pull/211013/commits/96c1bdf8fe9d122bbda86796c37688b103367a53
Btw. There is an eslint config that would throw an error in this scenario. I'm not sure if you have any concerns about it, but it's worth checking:
`'@typescript-eslint/consistent-type-imports': 'error',`

Reply:
ok, seems id didn't make a difference, thx for trying 👍

Reply:
Just wanted to say I think the 1KB increase per usage is expected. Unless a package is included in our shared packages, it's imported separately for each consumer and increases their bundles separately. I'm not sure why the increase to `securitySolution` is higher, but that seems to be common for that plugin, so I don't think it's something specific to this PR.

Reply:
Agreed, resolving